components/style/placementArrow.ts

```diff
@@ -84,166 +117,195 @@ export default function getArrowStyle<Token extends TokenWithCommonCls<AliasToke
       // ========================== Placement ==========================
       // Here handle the arrow position and rotate stuff
       // >>>>> Top
-      [[
-        `&-placement-top ${componentCls}-arrow`,
-        `&-placement-topLeft ${componentCls}-arrow`,
-        `&-placement-topRight ${componentCls}-arrow`,
-      ].join(',')]: {
-        bottom: 0,
-        transform: 'translateY(100%) rotate(180deg)',
-      },
+      ...isInject(arrowPlacement.top ?? true, {
```

Comment:
```suggestion
      ...isInject(arrowPlacement.top, {
```
`arrowPlacement` 的默认值不变，属性默认 false 就可以这样写了吧，

Reply:
done